src/pathfinding_service/model/channel_view.py

```diff
@@ -31,46 +32,35 @@ def __init__(
         self.partner = participant2
 
         self._deposit = deposit
-        self._transferred_amount = 0
-        self._received_amount = 0
-        self._locked_amount = 0
         self._capacity = deposit
         self.state = ChannelView.State.OPEN
         self.channel_id = channel_id
         self.settle_timeout = settle_timeout
         self.reveal_timeout = reveal_timeout
+        self.balance_update_nonce = balance_update_nonce
```

Comment:
Is there another `nonce`? Otherwise I'd prefer the shorter name.

Reply:
not yet, but there might be another nonce when we have withdraw and new deposits?!